src/components/user_view/edit_user_info/EditUserInfo.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from 'react';
+import Form from 'react-bootstrap/Form';
+import { FormInput } from '../../global/FormInput';
+import { Link, Redirect } from 'react-router-dom';
+
+export default class EditUserInfo extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
```

Comment:
Is there a reason we aren't letting users change their password?

Reply:
That's a good question. Do we want to let users change their password in this component, or should I create another one that handles password changes only and requires the user to confirm current password to do so?

Reply:
Confirming current password is probably a good idea. Also, it should ask for new password and confirm new password (similar to sign up page). This also might be useful if we ever decide to implement a "forgot password" feature...